Gemfile

```diff
@@ -26,7 +26,7 @@ end
 
 group :test do
   gem "ammeter"
-  gem "capybara", "3.35.3"
+  gem "capybara", "3.36.0"
```

Comment:
Why is this version locked in the Gemfile? Can we remove it?

Reply:
Great point. I've opened a PR to un-pin it upstream: https://github.com/thoughtbot/administrate/pull/2124
I'll pull that into our branch as well if it gets merged.